spec.html

```diff
@@ -532,198 +531,124 @@ <h1>InnerModuleEvaluation( _module_, _stack_, _index_ )</h1>
+          1. <ins>Perform ! Call(_module_.[[TopLevelCapability]].[[Resolve]], *undefined*, &laquo;*undefined*&raquo;).</ins>
+        1. <ins>Return *undefined*.</ins>
       </emu-alg>
     </emu-clause>
 
     <emu-clause id="sec-AsyncModuleExecutionRejected" aoid="AsyncModuleExecutionRejected">
       <h1><ins>AsyncModuleExecutionRejected ( _module_, _error_ )</ins></h1>
       <emu-alg>
-        1. Assert: _module_.[[Status]] is `"evaluated"`.
-        1. If _module_.[[AsyncEvaluating]] is *false*,
-          1. Assert: _module_.[[EvaluationError]] is not *undefined*.
-          1. Return *undefined*.
-        1. Assert: _module_.[[EvaluationError]] is *undefined*.
-        1. Set _module_.[[EvaluationError]] to ThrowCompletion(_error_).
-        1. Set _module_.[[AsyncEvaluating]] to *false*.
-        1. For each Module _m_ of _module_.[[AsyncParentModules]], do
-          1. If _module_.[[DFSIndex]] is not equal to _module_.[[DFSAncestorIndex]], then
-            1. Assert: _m_.[[DFSAncestorIndex]] is equal to _module_.[[DFSAncestorIndex]].
-          1. Perform ! AsyncModuleExecutionRejected(_m_, _error_).
-        1. If _module_.[[TopLevelCapability]] is not *undefined*, then
-          1. Assert: _module_.[[DFSIndex]] is equal to _module_.[[DFSAncestorIndex]].
-          1. Perform ! Call(_module_.[[TopLevelCapability]].[[Reject]], *undefined*, &laquo;_error_&raquo;).
-        1. Return *undefined*.
+        1. <ins>Assert: _module_.[[Status]] is `"evaluated"`.</ins>
+        1. <ins>If _module_.[[AsyncEvaluating]] is *false*,</ins>
+          1. <ins>Assert: _module_.[[EvaluationError]] is not *undefined*.</ins>
+          1. <ins>Return *undefined*.</ins>
+        1. <ins>Assert: _module_.[[EvaluationError]] is *undefined*.</ins>
+        1. <ins>Set _module_.[[EvaluationError]] to ThrowCompletion(_error_).</ins>
+        1. <ins>Set _module_.[[AsyncEvaluating]] to *false*.</ins>
+        1. <ins>For each Module _m_ of _module_.[[AsyncParentModules]], do</ins>
+          1. <ins>If _module_.[[DFSIndex]] is not equal to _module_.[[DFSAncestorIndex]], then</ins>
+            1. <ins>Assert: _m_.[[DFSAncestorIndex]] is equal to _module_.[[DFSAncestorIndex]].</ins>
+          1. <ins>Perform ! AsyncModuleExecutionRejected(_m_, _error_).</ins>
+        1. <ins>If _module_.[[TopLevelCapability]] is not *undefined*, then</ins>
+          1. <ins>Assert: _module_.[[DFSIndex]] is equal to _module_.[[DFSAncestorIndex]].</ins>
+          1. <ins>Perform ! Call(_module_.[[TopLevelCapability]].[[Reject]], *undefined*, &laquo;_error_&raquo;).</ins>
+        1. <ins>Return *undefined*.</ins>
       </emu-alg>
     </emu-clause>
   </emu-clause>
-</emu-clause>
 
-<emu-clause id="sec-source-text-module-records">
-    <h1>Source Text Module Records</h1>
-
-  <emu-clause id="sec-parsemodule" aoid="ParseModule">
-    <h1>ParseModule ( _sourceText_, _realm_, _hostDefined_ )</h1>
```

Comment:
I understand why you're moving this, but this part is impossible to review. Could you split the moving out into another commit? (Can still be in this PR.)

Reply:
Done